js/anims-missiles.js

```diff
@@ -108,18 +141,41 @@ function listenKeypressMissiles() {
         if (e.key in keys) {
             columnID = keys[e.key]
             isHit()
+            $city = $('#city-' +  columnID)
+            shootLaser($city)
         }
     })
 }
 
-/**
- * Increases animation speed based upon gameScore.score
- * We call this function inside repeatAnim function
+/*
+ * This changes the speed of the missiles based upon the player's current score.
  */
 function animationChangeSpeed() {
     return gameScore.score > 0 ? 3000 / (1 + (gameScore.score * 0.03)) : 3000
```

Comment:
technically thius means it will get infinitely faster, with no upper limit. i.e untill the missiles are falling every millisecond (1000 per second) is that what you want?

Reply:
Yes, because the game will start to get so hard the user will miss which will result in the score decreasing and thus the speed will go back to something the user is capable of dealing with.